pennylane/interfaces/torch.py

```diff
@@ -135,38 +170,47 @@ def forward(ctx, input_kwargs, *input_):
             return torch.from_numpy(res)
 
         @staticmethod
+        @once_differentiable
         def backward(ctx, grad_output):  # pragma: no cover
             """Implements the backwards pass QNode vector-Jacobian product"""
             # NOTE: This method is definitely tested by the `test_torch.py` test suite,
             # however does not show up in the coverage. This is likely due to
             # subtleties in the torch.autograd.FunctionMeta metaclass, specifically
             # the way in which the backward class is created on the fly
 
+            # determine the QNode variables which should be differentiated
+            diff_indices = None
+            non_diff_indices = set()
+
+            for arg, arg_variable in zip(ctx.saved_tensors, qnode.arg_vars):
```

Comment:
I'm less familiar with `ctx`, do we know that it will always be in the same order as `qnode.arg_vars`?

Reply:
Yep! The saved tensors and arg vars will always be in order of the positional arguments.